packages/dashboard/src/components/TfChainConnector.vue

```diff
@@ -277,11 +277,24 @@ export default class TfChainConnector extends Vue {
   public canLogin = typeof localStorage.getItem(key) === "string";
 
   public balance: any | null = null;
+  public twinID: any;
+  public grid: any;
+  @Watch("mnemonic")
+  async checkMnemonic() {
+    try {
+      this.grid = await getGrid(this.mnemonic);
+      this.grid.twinId;
+    } catch (err) {
+      this.mnemonicError = `Couldn't find a user for provided mnemonic on ${config.network} network.`;
```

Comment:
For better UX we can show 'Mnemonic is required' as an error message when the field is empty 
```suggestion
       if(!this.mnemonic) this.mnemonicError = "Mnemonic is required"
      else this.mnemonicError = `Couldn't find a user for provided mnemonic on ${config.network} network.`;
```